package.json

```diff
@@ -17,7 +19,13 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "gts": "^3.1.0",
```

Comment:
Do not touch my gts. I love it

Reply:
Not removed, just moved lower in the list of the dependencies.